.circleci/config.yml

```diff
@@ -92,6 +92,20 @@ aliases:
         only: main
 
 commands:
+  set-maximum-duration:
+    parameters:
+      seconds:
+        type: integer
+        default: 600
```

Comment:
There are some jobs that will take like 20 to 25 minutes I think? Should we make this 30 minutes? 🤔